concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-232	232	yellow	1424	YELLOW	296
+Seifart-2015-414-233	233	full	1429	FULL	297
+Seifart-2015-414-234	234	new	1231	NEW	298
+Seifart-2015-414-235	235	name	1405	NAME	301
+Seifart-2015-414-236	236	how	1239	HOW	302
+Seifart-2015-414-237	237	here	136	HERE	305
+Seifart-2015-414-238	238	there	1937	THERE	306
+Seifart-2015-414-239	239	other	197	OTHER	307
+Seifart-2015-414-240	240	fog	249	FOG	309
+Seifart-2015-414-241	241	flow	2003	FLOW	310
+Seifart-2015-414-242	242	sea	1474	SEA	311
+Seifart-2015-414-243	243	wet	1726	WET	312
+Seifart-2015-414-244	244	wash	1453	WASH	313
+Seifart-2015-414-245	245	worm	1219	WORM	314
+Seifart-2015-414-246	246	fur	580	FUR	316
+Seifart-2015-414-247	247	saliva	1503	SALIVA	318
+Seifart-2015-414-248	248	milk	635	MILK	319
+Seifart-2015-414-249	249	with	1340	WITH	320
+Seifart-2015-414-250	250	in	1460	IN	321
+Seifart-2015-414-251	251	at	1461	AT	322
+Seifart-2015-414-252	252	if	1459	IF	323
+Seifart-2015-414-253	253	child	2099	CHILD	327
+Seifart-2015-414-254	254	dark	706	DARK	328
+Seifart-2015-414-255	255	wide	1243	WIDE	330
+Seifart-2015-414-256	256	narrow	1267	NARROW	331
+Seifart-2015-414-257	257	thick	1244	THICK	334
+Seifart-2015-414-258	258	thin	2308	THIN	335
+Seifart-2015-414-259	259	short	1645	SHORT	336
+Seifart-2015-414-260	260	dull	379	BLUNT	338
+Seifart-2015-414-261	261	sharp	1396	SHARP	339
+Seifart-2015-414-262	262	dirty	1230	DIRTY	340
+Seifart-2015-414-263	263	rotten	1728	ROTTEN	341
+Seifart-2015-414-264	264	smooth	1234	SMOOTH	342
+Seifart-2015-414-265	265	straight	1404	STRAIGHT	343
+Seifart-2015-414-266	266	correct	1725	CORRECT (RIGHT)	344
+Seifart-2015-414-267	267	left	244	LEFT	345
+Seifart-2015-414-268	268	right	1019	RIGHT	346
+Seifart-2015-414-269	269	old	1229	OLD	347
+Seifart-2015-414-270	270	pull	1455	PULL	349
+Seifart-2015-414-271	271	throw	1456	THROW	351
+Seifart-2015-414-272	272	hit	1433	HIT	352
+Seifart-2015-414-273	273	split	1437	SPLIT	353
+Seifart-2015-414-274	274	pierce	398	PIERCE	354
+Seifart-2015-414-275	275	dig	1418	DIG	355
+Seifart-2015-414-276	276	tie	1917	TIE	356
+Seifart-2015-414-277	277	fall	1280	FALL	358
+Seifart-2015-414-278	278	swell	1573	SWELL	359
+Seifart-2015-414-279	279	think	2271	THINK	360
+Seifart-2015-414-280	280	sing	1261	SING	361
+Seifart-2015-414-281	281	smell	2124	SMELL	362
+Seifart-2015-414-282	282	blow	175	BLOW (OF WIND)	365
+Seifart-2015-414-283	283	fear	1419	FEAR (BE AFRAID)	366
+Seifart-2015-414-284	284	squeeze	1414	SQUEEZE	367
+Seifart-2015-414-285	285	hold	1448	HOLD	368
+Seifart-2015-414-286	286	down	1326	DOWN	369
+Seifart-2015-414-287	287	up	1591	UP	370
+Seifart-2015-414-288	288	ripe	178	RIPE	371
+Seifart-2015-414-289	289	dust	2	DUST	372
+Seifart-2015-414-290	290	alive	1422	BE ALIVE	373
+Seifart-2015-414-291	291	rope	1218	ROPE	374
+Seifart-2015-414-292	292	year	1226	YEAR	375
+Seifart-2015-414-293	293	diminutive			378
+Seifart-2015-414-294	294	big river			379
+Seifart-2015-414-295	295	longhouse			380
+Seifart-2015-414-296	296	stone axe	677	AXE	382
```

Comment:
We can unite this with "steel axe", since they present the same forms. See note in the original concept list